tools/topology/topology2/include/components/src_format_s32_convert_from_48k.conf

```diff
@@ -1,94 +1,26 @@
 #src format array
 			num_output_audio_formats 14
 
-			Object.Base.output_audio_format [
-				# 8khz output
-				{
-					out_rate                 8000
-					out_bit_depth            32
-					out_valid_bit_depth      32
-				}
-				# 11.025 khz output
-				{
-					out_rate                 11025
-					out_bit_depth            32
-					out_valid_bit_depth      32
-					obs "$[($out_channels * (($[($out_rate + 999)] / 1000) + 4)) * ($out_bit_depth / 8)]"
-				}
-				# 12khz output
-				{
-					out_rate                 12000
-					out_bit_depth            32
-					out_valid_bit_depth      32
-				}
-				# 16khz output
-				{
-					out_rate                 16000
-					out_bit_depth            32
-					out_valid_bit_depth      32
-				}
-				# 22.05khz output
-				{
-					out_rate                 22050
-					out_bit_depth            32
-					out_valid_bit_depth      32
-				}
-				# 24khz output
-				{
-					out_rate                 24000
-					out_bit_depth            32
-					out_valid_bit_depth      32
-				}
-				# 32khz output
-				{
-					out_rate                 32000
-					out_bit_depth            32
-					out_valid_bit_depth      32
-				}
-				# 44.1khz output
-				{
-					out_rate                 44100
-					out_bit_depth            32
-					out_valid_bit_depth      32
-					obs "$[($out_channels * (($[($out_rate + 999)] / 1000) + 4)) * ($out_bit_depth / 8)]"
-				}
-				# 48khz output
-				{
-					out_rate                 48000
-					out_bit_depth            32
-					out_valid_bit_depth      32
-				}
-				# 64khz output
-				{
-					out_rate                 64000
-					out_bit_depth            32
-					out_valid_bit_depth      32
-				}
-				# 88.2khz output
-				{
-					out_rate                 88200
-					out_bit_depth            32
-					out_valid_bit_depth      32
-					obs "$[($out_channels * (($[($out_rate + 999)] / 1000) + 4)) * ($out_bit_depth / 8)]"
-				}
-				# 96khz output
-				{
-					out_rate                 96000
-					out_bit_depth            32
-					out_valid_bit_depth      32
-				}
-				# 176.4khz output
-				{
-					out_rate                 176400
-					out_bit_depth            32
-					out_valid_bit_depth      32
-					obs "$[($out_channels * (($[($out_rate + 999)] / 1000) + 4)) * ($out_bit_depth / 8)]"
```

Comment:
some of these rates have an obs set that you seem to have removed @btian1

Reply:
sorry, missed this part.

Reply:
@btian1 @ranj063 btw, obs is changing name to be more meaningful. Please check in with @marcinszkudlinski for details.

Reply:
@lgirdwood @marcinszkudlinski , is it name change or calculation change different with current implementation?
we can use a separate patch to address this change.

Reply:
@btian1 sorry but what you have now is also wrong. The new value of obs is only set for 3 of the frequencies in the original file and not for all frequencies.